src/armory/Armory.js

```diff
@@ -281,48 +280,119 @@ class Armory extends React.Component<Props, State> {
 		});
 	};
 
-
+	//The || in the values are for a warning read more here https://github.com/mui-org/material-ui/issues/4904
 	render(): React.Node {
 		return (
 			<div id="Parent">
 				<Navbar styleName="navbtn" linkName="MainMenu" returnName="Back to Main Menu" pageName="Armory" userName="FRIcker | $465128" />
-				<div className="column armoryleft">
-					<h3>Select a Tank to Edit</h3>
-					<select className="dropdownMenu" value={this.state.selectedTankId} onChange={this.handleChangeInTankOptions}>{tankOptions.map(({ value, label }, index) => <option key={index}  value={value}>{label}</option>)}</select>
 					<div className="column armoryleft">
+						<h3>Select a Tank to Edit</h3>
+						<select 
+							className="dropdownMenu" 
+							value={this.state.selectedTankId} 
+							onChange={this.handleChangeInTankOptions}
+						>
+							{tankOptions.map(({ value, label }, index) => <option key={index}  value={value}>{label}</option>)}
+						</select>
 						<h6>Set this tank as default?</h6>
-						<button type="button" className="btn">Set Default</button>
+						<button type="button" className="btn mb-4">Set Default</button>
+						<CreateNewTankPopup ref="CreateNewTankPopup"/>
 						<h3>Edit tank's Code</h3>
 						<Link to="Casus">
-						<button type="button" className="btn">Casus</button>
+							<button type="button" className="btn">Casus</button>
 						</Link>
 					</div>
 					<div className="column armorymiddle">
-						<h1>BIG TANK GUY</h1>
+						<h1>{this.state.selectedTankName}</h1>
 						<h6>Points Used: 0/10</h6>
 					</div>
 					<div className="column armoryright">
 						<h6>Chassis</h6>
-						<select className="tankComponentMenu" value={this.state.selectedChassis} onChange={this.handleChangeInChassisOptions}>{chassisOptions.map(({ value, label  }, index) => <option key={index} value={value}>{label}</option>)}</select>
+						<select 
+							className="tankComponentMenu"
+							value={this.state.selectedChassis || ''}
```

Comment:
Talked about offline, but make sure you delete this `|| ''` and all similar ones before pushing